.github/workflows/pin-build-tools-image.yml

```diff
@@ -33,6 +33,7 @@ concurrency:
 # No permission for GITHUB_TOKEN by default; the **minimal required** set of permissions should be granted in each job.
 permissions: {}
 
+# only used in check-manifests! tag-image has these inlined, because the `env` context isn't available in `jobs.<job_id>.with.<with_id>`, preventing us from using these variables there.
 env:
   FROM_TAG: ${{ inputs.from-tag }}
   TO_TAG: pinned
```

Comment:
Btw, if it's only for one job/step, I think we can move it in inside the job/step